manifests/k/ktsu/BuildMonitor/1.0.0.0/ktsu.BuildMonitor.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ktsu.BuildMonitor
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: ktsu
+PackageName: BuildMonitor
+License: MIT License
+Copyright: ktsu © 2024
+ShortDescription: ktsu.io.BuildMonitor

```

Comment:
Please provide a better description of what this application is, what it does, or what it is used for

Reply:
Addressed in: https://github.com/microsoft/winget-pkgs/pull/160177/commits/df3cf40664555290d5783b9d2c7702ecc42bf3c2